pypsrp/complex_objects.py

```diff
@@ -110,7 +110,9 @@ def __init__(self):
         self.types = []
 
     def __str__(self):
-        return to_string(self.to_string)
+        return to_string(
+            self.to_string if self.to_string is not None else 'None'
```

Comment:
Just wondering whether this should be `None` or just `""`? Either option is fine with me and I'm just wondering if you had a particular preference to `None` or that's just what you chose.

Reply:
Just because `str(None)` is `"None"` and not an empty string.

Reply:
Good point, I'm happy with `None`.